apps/site/pages/docs/getting-started/getting-started-overview.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: ' Getting Started Overview'
+sidebar_label: 'Getting Started'
+---
+
+import { Callout } from 'nextra-theme-docs'
+
+<header>
+
+# Getting Started Overview
+
+</header>
+
+<Callout type="warning" emoji="⚠️">
+This documentation is currently under development.
+</Callout>
+
+In this Getting Started track, you'll build and deploy a FastStore Storefront at the domain `https://account-name.myvtex.com/`.
+
+By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS](/tbd), [Catalog](/tbd), and [Checkout](/tbd).
+Here's an example of the final result: [Starter Store](/tbd).
```

Comment:
```suggestion
Here's an example of the final result: [Starter Store](https://starter.vtex.app).
```

Are we linking to the repo or the application? 🤔

Reply:
To the application Fanny :)